magicctapipe/io/io.py

```diff
@@ -528,6 +735,92 @@ def load_train_data_files(
 
     return data_train
 
+def load_train_data_files_tel(input_dir, config, offaxis_min=None, offaxis_max=None, true_event_class=None):
+    """
+    Loads DL1-stereo data files and separates the shower events per
+    telescope combination type for training RFs.
+
+    Parameters
+    ----------
+    input_dir: str
+        Path to a directory where input DL1-stereo files are stored
+    config: dict 
+        yaml file with information about the telescope IDs. 
+    offaxis_min: str
+        Minimum shower off-axis angle allowed, whose format should be
+        acceptable by `astropy.units.quantity.Quantity`
+    offaxis_max: str
+        Maximum shower off-axis angle allowed, whose format should be
+        acceptable by `astropy.units.quantity.Quantity`
+    true_event_class: int
+        True event class of the input events
+    
+
+    Returns
+    -------
+    data_train: dict
+        Data frames of the shower events separated telescope-wise
+
+
+    Raises
+    ------
+    FileNotFoundError
+        If any DL1-stereo data files are not found in the input
+        directory
+    """
+    
+    TEL_NAMES, _ = telescope_combinations(config)
```

Comment:
hmm, I think training was also included in the test, so I'm wondering why codecov complains that this part of the code is not checked by the tests.

Reply:
This is a telescope-wise training (no combo type). But we can add a test, which will be needed by the next PR

Reply:
I think you even over did it with the MAGIC-only tests, since you are testing the same function twice, but fine to keep it as it is.